### DISCUSSION TITLE
blog: Is it possible to change the order of Archive or collapse it?

### DISCUSSION THREAD

**Discussion by mfridman** [squidfunk/mkdocs-material#7019] [Q&A] [answered]
I have both Archive and Category sections, however, as the number of years increases it pushes the Category down to the point it may not be visible. 

Is it possible to collapse _only_ the Archive section by default? Or is it possible to swap the Archive and Category positions?

![CleanShot 2024-04-06 at 09 01 22@2x](https://github.com/squidfunk/mkdocs-material/assets/6278244/7571e80b-85fa-4a53-b7cc-3ad8ae29001d)

**Comment by squidfunk** [comment, marked as answer]
Thanks for asking. It's currently not possible to influence the position of the archive or category in the navigation, but we can check if we can somehow make that happen. Would you like to create a feature request for this?

**Reply by squidfunk** [reply]
Addendum: regarding collapsing – do you have `navigation.expand` enabled? It should otherwise be collapsed by default.

**Reply by mfridman** [reply]
~Yes, I have it enabled, the behaviour I observed was the opposite though when I had it disabled .. **Archive** was expanded and **Category** was collapsed.~

Nvm, it works as expected (I had my page open in a post).

**Comment by mfridman** [comment]
Let me noddle on this a bit before filing a feature request, I don't want to waste your time and make sure I thought through whether this is specific to my use case or this is generally useful. I.e., something like `archive_collapsed: true|false` or `category_collapsed: true|false`.

**Reply by squidfunk** [reply]
Settings like this would not integrate well with the way how navigation works. We could allow to move the archive and category section somewhere else, but we cannot auto-collapse/expand single sections with the current design.
